### DISCUSSION TITLE
GQL: Load from Introspection

### DISCUSSION THREAD

**Discussion by cirelli94** [usebruno/bruno#3604] [Q&A] [answered]
I'm trying to load the GQL schema for the documentation using the `Load from Introspection` button but I got this error:
![image](https://github.com/user-attachments/assets/d5d80888-8ae1-4030-a26d-038e0eecda3b)
I'm using an url with a {{baseUrl}} variable, when I replace the {{baseUrl}} with the read value I don't get the error and it works.

Is this a bug or I'm doing something wrong?

Bruno v1.36.1

**Comment by Its-treason** [comment, marked as answer]
This looks like a bug. Variables are not correctly interpolated when loading the schema from introspection. Could you open a bug report? https://github.com/usebruno/bruno/issues/new?assignees=&labels=bug&projects=&template=BugReport.yaml
